dna/course/zomes/courses/code/src/lib.rs

```diff
@@ -102,11 +102,84 @@ mod courses {
         course::handlers::get_my_enrolled_courses()
     }
 
-    // Section
+    //  ====================== Section definitions
     // TODO: implement section entry definitions
-    // TODO: implement section CRUD methods
+    #[entry_def]
+    fn section_entry_definition() -> ValidatingEntryType {
+        course::entry::section_entry_def()
+    }
+
+    #[entry_def]
+    fn section_anchor_definition() -> ValidatingEntryType {
+        course::anchor::section_anchor_def()
+    }
+
+    // TODO: implement section CRUD methods --> fished out of modules "impl" right? or handlers?
+    #[zome_fn("hc_public")]
+    fn create_section(title: String, course_anchor_address: Address, timestamp: u64, anchor_address: Address,
+    ) -> ZomeApiResult<Address> {
+        section::handlers::create(title, course_anchor_address, timestamp, anchor_address)
+    }
 
-    // Content
+    #[zome_fn("hc_public")]
+    fn get_latest_section_entry(
+        section_anchor_address: Address,
+    ) -> ZomeApiResult<Option<section::entry::Section>> {
+        section::handlers::get_latest_section_entry(section_anchor_address)?;
+    }
+    // this is copied off the course methods code and changed... is the sections_address part relevant here or in the above code?
+    #[zome_fn("hc_public")]
+    fn update_section(
+        title: String,
+        course_anchor_address: Address,
+        timestamp: u64,
+        anchor_address: Address,
+    ) -> ZomeApiResult<Address> {
+        section::handlers::update(title, course_anchor_address, timestamp, anchor_address)
+    }
+
+    #[zome_fn("hc_public")]
+    fn delete_section(section_anchor_address: Address) -> ZomeApiResult<Address> {
+        section::handlers::delete(section_anchor_address)
+    }
```

Comment:
With the explanation nastasia gave us last session, we would need to add timestamp in delete_section and update the timestamp of the course entry so that we will not be ending up having duplicate course entries (which causes the bug nastasia mentioned last session :D)

```suggestion
    fn delete_section(section_anchor_address: Address, timestamp: u64) -> ZomeApiResult<Address> {
        section::handlers::delete(section_anchor_address, timestamp)
    }
```